addons/l10n_it_edi/models/account_move.py

```diff
@@ -540,7 +536,7 @@ def _l10n_it_edi_features_for_document_type_selection(self):
             'downpayment': self._is_downpayment(),
             'services_or_goods': services_or_goods,
             'goods_in_italy': services_or_goods == 'consu' and self._l10n_it_edi_goods_in_italy(),
-            'fee_statement': self._l10n_it_edi_is_fee_statement(),  # For downpayment
+            'professional fees': self._l10n_it_edi_is_professional_fees(),
```

Comment:
Not sure having a space in a dict key is a good idea :nerd_face:

Reply:
Yep you are right :grimacing: will do a fix pr to be extra safe